### PR TITLE
Update config to new Samsung TV integration

### PR DESCRIPTION
**Pull request by escoand**
**Description:**
Update description to new configuration options.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#28306

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/samsungtv.markdown [hunk 1]
```diff
@@ -20,39 +20,33 @@ To add a TV to your installation without relying on the [discovery component](/i
 
 ```yaml
 # Example configuration.yaml entry
-media_player:
-  - platform: samsungtv
-    host: IP_ADDRESS
+samsungtv:
+  - host: IP_ADDRESS
+    name: My Samsung TV
```
Comment (klaasnicolaas) [inline source/_integrations/samsungtv.markdown]: In the first example we want only required config variables, please remove everything that is optional here.
You can make a more advanced example above the `supported models` part 😉

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
@escoand My main concern with this PR right now, is that it has no mention of configuring/setting up the integration via the integrations panel in the frontend.

I think that should be added. Also, add the `ha_config_flow: true` to the front matter of the page, to indicate this integration has a config flow.

**Comment by frenck** [comment]
Maybe I'm missing something, but you can go to the integrations panel and just add the integration manually, right?

**Comment by escoand** [comment]
Correct, as I wrote in line 18:

```markdown
You can add your TV by using the web interface via [ssdp](/integrations/ssdp) discovery or manual config.
For more control over the integration you can also use an entry in the configuration files.
```

Maybe this is a bit misleading as I used "manual config" also for it.

**Comment by frenck** [comment]
That does not really want I meant with my original comment.

There is a manual setup (click path) for the user via integrations to set it up manually. (Click integrations -> add using the + -> etc.

The manual should be written from a perspective a user of any experience level can use it. Right now, I think we should add instructions on how to set it up via just the frontend (since that is now possible).

**Comment by arsaboo** [comment]
Since the component now allows both config flow and yaml, maybe the [Ring docs](https://github.com/home-assistant/home-assistant.io/pull/11679) are a good reference.

**Comment by escoand** [comment]
OK, what do you think about he new wording?

**Review by frenck — APPROVED** [review]
Thanks!

**Pull request merged**